### DISCUSSION TITLE
Question about two column layouts in Rmarkdown

### DISCUSSION THREAD

**Discussion by Rendan86** [BryanJenksCommunity/FAQ#43] [Q&A]
Hi Bryan,

I come from this video in youtube
https://www.youtube.com/watch?v=qLCVcI7feMM&ab_channel=BryanJenks

I loved your explanation, and wanted to go with your roundabaout with the CSS trick, but althouhg I literally copied your code in the video I did not manage to replicate the result... I would prefer to apply this approach instead of the tufte, do you have any idea of what I could be missing?
I am using R and Rstudio updated to the latest version.

**Comment by tallguyjenks** [comment]
okay, cool, thank you for bringing this here :)
Can you please post the code you're using at the moment that is not working correctly?
